crates/evm/src/tests/hooks_tests.rs

```diff
@@ -47,7 +53,8 @@ fn end_soft_confirmation_hook_sets_head() {
 
     evm.begin_soft_confirmation_hook(
         DA_ROOT_HASH.0,
-        GENESIS_STATE_ROOT.as_ref(),
+        pre_state_root,
+        &pre_state_root,
```

Comment:
please define a constant called `DA_TX_COMMITMENT` and use that 
`pre_state_root` for both args might be confusing for readers